modules/GoogleDialer/Android.mk

```diff
@@ -9,3 +9,14 @@ GAPPS_LOCAL_OVERRIDES_MIN_VARIANT :=
 GAPPS_LOCAL_OVERRIDES_PACKAGES := Dialer
 
 include $(BUILD_GAPPS_PREBUILT_APK)
```

Comment:
minor suggestion:  maybe add a blank line or a comment before CLEAR_VARS so it is more obvious that there are actually two modules defined in this file.